pyproject.toml

```diff
@@ -146,17 +147,27 @@ legacy_tox_ini = """
 [tox]
 isolated_build = true
 envlist =
-    py{39,310,311}
-    style
+    py{39,310,311}-pytest
     py{39,310,311}-typing
+    docs
```

Comment:
docs needed here, otherwise it is not run

Reply:
Hmm okay, really thought I tested it and did not need it, but if you say so